bin/ocaml_merlin.ml

```diff
@@ -16,12 +16,24 @@ let man =
 let info = Term.info "ocaml-merlin" ~doc ~man
 
 let term =
-  let+ common = Common.term in
+  let+ common = Common.term
+  and+ dump_config =
+    Arg.(
+      value
+      & opt (some string) None
+      & info [ "dump-config" ]
+          ~doc:
+            "Print the entire content of the merlin configuration in the \
+             given folder in the form of a S-expression.")
```

Comment:
We now have a user-friendly output, contrary to what the description says. Might be worth changing the description to instead say that this command doesn't provide stability guarantee with a clear "Disclaimer", so that users are warned if they try to build systems on top of it.